### ISSUE TITLE
*TESTS* Systolic blood pressure reading

### ISSUE THREAD

**Issue by richiecroker** [opensafely/codelist-development#80]
https://codelists.opensafely.org/codelist/user/richard-croker/systolic-blood-pressure-tests/732cb8c9/

A number of codes here which could possibly be removed (sitting, standing etc).  Ambulatory codes have been excluded for now.  All for discussion.

**Comment by richiecroker** [comment]
@brianmackenna looking at the full codelist, suggest we don't need separate numerical value version for this sentinel test.

**Comment by richiecroker** [comment]
but updated QoF SBP codelist here: https://codelists.opensafely.org/codelist/user/richard-croker/systolic-blood-pressure-qof/3572b5fb/

**Comment by brianmackenna** [comment]
Looks good to me. Just to say that @inglesp is importing all QOF codelists soon and this will be essentially a duplicate then - we should possibly remove at that point and default to QoF, which presumably we will update at regular intervals in the future.

**Comment by richiecroker** [comment]
No, because this specifically relates to _systolic_ blood pressure - the QOF codelists contain both systolic and diastolic

**Comment by brianmackenna** [comment]
Looks good to me.
